### ISSUE TITLE
Replace (@cite) in REPL help mode. 

### ISSUE THREAD

**Issue by cmichelenstrofer** [ali-ramadhan/DocumenterCitations.jl#62]
Markdown links `[some text](URL)` get displayed as hyperlinks in the created document (HTML) but in text-based outputs like the REPL help (?) mode it gets substituted by `some text (URL)`. Similarly `(@ref)`s get deleted. It would be nice to have something similar and either remove the (@cite) or format it some other way.

**Comment by navidcy** [comment]
Yes! I was about to make an issue for this and I saw this issue. I vote for this! I think a replacement with either `Authors (Year); doi:XXX.XXX.XXX` or just `Authors (Year); url:https://.........` or just `Authors (Year)` if no doi/url are provided.

**Comment by goerz** [comment]
Now tracked at https://github.com/JuliaDocs/DocumenterCitations.jl/issues/21
